src/IO/FileFormat/FileJPG.cpp

```diff
@@ -39,7 +39,7 @@ bool ReadImageFromJPG(const std::string &filename, Image &image)
 	JSAMPARRAY buffer;
 
 	if ((file_in = fopen(filename.c_str(), "rb")) == NULL) {
-		PrintWarning("Read JPG failed: unable to open file.\n");
+		PrintWarning("Read JPG failed: unable to open file: '%s'.\n", filename.c_str());
```

Comment:
do we need `'%s'` instead of `%s`?
and lets remove period as it will display like `Read JPG failed: unable to open file: image.name.`

Reply:
quotes and dots are removed.